deployment/keystone/changeset/append_node_capbilities.go

```diff
@@ -16,15 +20,39 @@ type AppendNodeCapabilitiesRequest = MutateNodeCapabilitiesRequest
 // AppendNodeCapabilities adds any new capabilities to the registry, merges the new capabilities with the existing capabilities
 // of the node, and updates the nodes in the registry host the union of the new and existing capabilities.
 func AppendNodeCapabilities(env deployment.Environment, req *AppendNodeCapabilitiesRequest) (deployment.ChangesetOutput, error) {
-	cfg, err := req.convert(env)
+	c, err := req.convert(env)
 	if err != nil {
 		return deployment.ChangesetOutput{}, err
 	}
-	_, err = internal.AppendNodeCapabilitiesImpl(env.Logger, cfg)
+	r, err := internal.AppendNodeCapabilitiesImpl(env.Logger, c)
 	if err != nil {
 		return deployment.ChangesetOutput{}, err
 	}
-	return deployment.ChangesetOutput{}, nil
+	out := deployment.ChangesetOutput{}
+	if req.UseMCMS {
+		if r.Ops == nil {
+			return out, fmt.Errorf("expected MCMS operation to be non-nil")
+		}
+		timelocksPerChain := map[uint64]common.Address{
+			c.Chain.Selector: c.ContractSet.Timelock.Address(),
+		}
+		proposerMCMSes := map[uint64]*gethwrappers.ManyChainMultiSig{
+			c.Chain.Selector: c.ContractSet.ProposerMcm,
+		}
+
+		proposal, err := proposalutils.BuildProposalFromBatches(
+			timelocksPerChain,
+			proposerMCMSes,
+			[]timelock.BatchChainOperation{*r.Ops},
+			"proposal to set update node capabilities",
+			0,
```

Comment:
probably worth parameterizing this just in case you want to add timelock delay in the future. @carte7000 had a good idea here https://github.com/smartcontractkit/chainlink/pull/15525/files#diff-f6eeb14804b5d16769abc3f48ad18187965263d5e5d29345e4079e85199de731R38 to use an MCMS struct being nil/not-nil as a way to signal that but also keep the door open for any other MCMS specific params